ConsoleApp1/VoteSystem.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System.Linq;
+
+namespace ConsoleApp1
+{
+	public interface IVoteRepository
+	{
+		ICollection<Vote> GetAllVotes();
+
+		/// <summary>
+		/// Finds vote by title
+		/// </summary>
+		/// <param name="title">Vote title</param>
+		/// <returns>Vote or null if not found</returns>
+		Vote GetVoteByTitle(string title);
+	}
+
+
+	public class VoteSystem
+	{
+		private readonly IVotePromptService votePromptService;
+		private readonly IVoteRepository voteRepository;
+
+		public IList<Vote> Votes { get; }
+
+		public VoteSystem(IVotePromptService votePromptService, IVoteRepository voteRepository)
+		{
+			this.votePromptService = votePromptService;
+			this.voteRepository = voteRepository;
+			Votes = null;
+		}
+
+		public void ShowVote(int voteId)
+		{
+			string key = string.Empty;
+			Console.WriteLine($"Title: {Votes[voteId].Title}");
+			foreach (var question in Votes[voteId].Questions)
+			{
+				Console.WriteLine($"Питання: {question.Title}");
+				Console.WriteLine("Варіанти:");
+				for (int i = 0; i < question.Answers.Count; i++)
+				{
+					Console.WriteLine($"{i}.{question.Answers[i].Title}");
+				}
+
+				int answerId;
+				do
```

Comment:
Voting is not responsibility of `ShowVote` method
Vote should accept voteId, you should delegate prompting operation to someone who is responsible for it